vignettes/git_usage.Rmd

```diff
@@ -96,16 +99,25 @@ Once you have completed the Pull Request you will see all committed changes are
 
 ### Reviewing/Closing an Issue
 
-- At least one reviewer must approve the Pull Request. Please review the [Pull Request Review Guidance](pr_review_guidance.html), which provides in depth guidance on doing a proper Pull Request. 
-- The reviewer must ensure that the function follows the programming strategy recommendations. 
-- Any comment/question/discussion must be addressed and documented in GitHub before the Pull Request is merged and the issue closed.
+- At least one reviewer must approve the Pull Request. Please review the [Pull
+Request Review Guidance](pr_review_guidance.html), which provides in depth
+guidance on doing a proper Pull Request.
+- The reviewer must ensure that the function follows the programming strategy
+recommendations.
+- Any comment/question/discussion must be addressed and documented in GitHub
+before the Pull Request is merged
 
-Once the review is completed, the reviewer will merge the Pull Request and click **Delete branch** in the Pull Request.
+Once the review is completed, the reviewer will merge the Pull Request and click
+**Delete branch** in the Pull Request.
 
 ```{r, echo = FALSE}
     knitr::include_graphics("github_delete_branch.png", dpi = 144)
 ```
 
+After merging the Pull Request the corresponding issue must be moved to the
+"Sprint Done" column on the "admiral scrum board". The issue must not be closed.
+It will be closed automatically once the `devel` branch is merged to the `main`
+branch.
```

Comment:
Do we want to specify who is supposed to do this?  The Reviewer, the person who made the PR, the scrum master?

Reply:
Also do we want an image or two here showing the Project board and moving stuff?  Umm...now I'm nervous about people accessing the board and messing things up??

Reply:
I have added the responsible person and a screenshot.

You can move the issue without going to the project board. This should avoid messing up the board.